dev/com.ibm.ws.concurrent_fat_cdi/fat/src/com/ibm/ws/concurrent/cdi/fat/ConcurrentCDITest.java

```diff
@@ -41,22 +42,27 @@ public class ConcurrentCDITest extends FATServletClient {
 
     public static final String APP_NAME = "concurrentCDIApp";
     public static final String APP_NAME_EE10 = "concurrentCDI4App";
+    public static final String APP_NAME_EE10_2 = "concurrentCDIWeb2";
```

Comment:
Should this be named to use the same convention as other application artifacts in the bucket, where the targeted CDI specification level is included in the name?  If I followed the intent correctly,
```suggestion
    public static final String WEBAPP_NAME_EE10 = "concurrentCDI4WebApp";
```
If changing this, some other places would need updates accordingly